doc/manual/variables-and-scoping.rst

```diff
@@ -386,24 +445,29 @@ existing value location, and ``let`` creates new locations. This
 difference is usually not important, and is only detectable in the
 case of variables that outlive their scope via closures. The ``let``
 syntax accepts a comma-separated series of assignments and variable
-names::
+names:
 
-    let var1 = value1, var2, var3 = value3
-        code
-    end
+.. doctest::
+
+    julia> let x = 1, y = 3, z = 4
```

Comment:
Ah, I missed the intent of the original example and thought it was an error. I changed the example at line 452 to the one you've given here.